composer.json

```diff
@@ -16,8 +16,9 @@
     ],
     "require": {
         "php": ">=5.3.3",
-        "rollerworks/search": ">=1.0.0-beta4@dev",
-        "symfony/validator": "~2.3"
+        "rollerworks/search": ">1.0.0-beta4@dev",
+        "symfony/validator": "^2.3.3",
```

Comment:
I have no idea :) this is how it used bot phpspec and it works.

`~2.3` is allowed. But I need at least 2.3.3 as anything lower has some serious bugs.